api/tests/core/educational/api/test_move_collective_offer.py

```diff
@@ -62,6 +62,58 @@ def create_offer_by_booking_state(venue, state):
 
 
 class MoveCollectiveOfferSuccessTest:
+    def test_move_collective_offer_with_its_own_OA(self):
+        """
+        Ensure the collective offer with its own OA keeps it once moved.
+        """
+        collective_offer = educational_factories.CollectiveOfferOnOtherAddressLocationFactory()
+        destination_venue = offerers_factories.VenueFactory(managingOfferer=collective_offer.venue.managingOfferer)
+        source_venue = collective_offer.venue
+        assert collective_offer.offererAddress != source_venue.offererAddress
+        offer_OA_id = collective_offer.offererAddress.id
```

Comment:
(détail pour la compréhension)
```suggestion
        offer_OA_id = collective_offer.offererAddressId
```